docs/documentation/functions/certificates/Invoke-RequestSignedCertificate.md

```diff
@@ -0,0 +1,225 @@
+# Invoke-RequestSignedCertificate
+
+## Synopsis
+
+Request Certificate Signing Request (CSR) to be signed by a Certificate Authority.
+
+## Syntax
+
+```powershell
+Invoke-RequestSignedCertificate [-csrFilePath] <String> [-outDirPath] <String> [-certificateAuthority] <String>
+[-caKeyPath] <String> [-caCertPath] <String> [-expireDays] <Int> [-caFqdn] <String> [-username] <String> 
+[-password] <String> [-certificateTemplate] <String> [-getCArootCert] <Switch>
+```
+
+## Description
+
+The `Invoke-RequestSignedCertificate` cmdlet signs off a certificate signing request (CSR) using either the Microsoft Certificate Authority web enrollment service or OpenSSL method.
+
+## Examples
+
+### Example 1
+
+```powershell
+Invoke-RequestSignedCertificate -csrFilePath "C:\certificates\sfo-vrli01.rainpole.io.csr" -outDirPath "C:\certificates" -certificateAuthority "msca" -caFqdn "rpl-ad01.rainpole.io" -username "Administrator" -password "VMw@re1!" -certificateTemplate "VMware"
+```
+
+This example will request the sfo-vrli01 Certificate Signing Request (CSR) file to be signed by the Certificate Authority rpl-ad01.rainpole.io.
+
+### Example 2
+
+```powershell
+Invoke-RequestSignedCertificate -csrFilePath "C:\certificates\sfo-vrli01.rainpole.io.csr" -outDirPath "C:\certificates" -certificateAuthority "msca" -caFqdn "rpl-ad01.rainpole.io" -username "Administrator" -password "VMw@re1!" -certificateTemplate "VMware" -getCArootCert
```

Comment:
Should these examples be sfo.rainmpole.io?

Reply:
addressed in commit c96a0aa15eab993f1b7f284e5e402712b1a9401f